src/kindlemint/generators/sudoku_generator.py

```
@@ -0,0 +1,154 @@
+        
+        # Remove numbers randomly
+        positions = [(i, j) for i in range(9) for j in range(9)]
+        random.shuffle(positions)
+        
+        removed = 0
+        for row, col in positions:
+            if removed >= remove_count:
+                break
+            
+            # Try removing this number
+            original = puzzle[row][col]
```

Ruff (0.11.9)

Local variable `original` is assigned to but never used

Remove assignment to unused variable `original`

(F841)